openpype/modules/deadline/plugins/publish/submit_publish_job.py

```diff
@@ -446,12 +447,15 @@ def _create_instances_for_aov(self, instance_data, exp_files):
             app = os.environ.get("AVALON_APP", "")
 
             preview = False
-            if app in self.aov_filter.keys():
-                for aov_pattern in self.aov_filter[app]:
-                    if re.match(aov_pattern, aov):
-                        preview = True
-                        break
 
+            if isinstance(col, list):
+                render_file_name = os.path.basename(col[0])
+            else:
+                render_file_name = os.path.basename(col)
+            aov_patterns = self.aov_filter.keys()
+            preview = match_aov_pattern(app, aov_patterns, render_file_name)
```

Comment:
I don't think we should be passing just the keys to the function. This seems like a bug?

Reply:
Thank you it might be, will take a look in a bit

Reply:
Yep you are right, definite bug I'm looking into it